src/fswAlgorithms/attGuidance/oneAxisSolarArrayPoint/oneAxisSolarArrayPoint.c

```diff
@@ -48,7 +48,52 @@ void SelfInit_oneAxisSolarArrayPoint(oneAxisSolarArrayPointConfig *configData, i
 */
 void Reset_oneAxisSolarArrayPoint(oneAxisSolarArrayPointConfig *configData, uint64_t callTime, int64_t moduleID)
 {
+    if (!NavAttMsg_C_isLinked(&configData->attNavInMsg)) {
+        _bskLog(configData->bskLogger, BSK_ERROR, " oneAxisSolarArrayPoint.attNavInMsg wasn't connected.");
+    }
 
+    // check how the input body heading is provided
+    if (BodyHeadingMsg_C_isLinked(&configData->bodyHeadingInMsg)) {
+        configData->bodyAxisInput = inputBodyHeadingMsg;
+    }
+    else {
```

Comment:
:small_red_triangle_down: This additional set of braces is not needed. Simply do if message else if htHat else Error

Reply:
I'm not sure I'm understanding this one. I might need to talk quickly whenever you're available.